Tests/DataCollector/DeviceDataCollectorTest.php

```diff
@@ -27,7 +27,7 @@
  *
  * @author suncat2000 <nikolay.kotovsky@gmail.com>
  */
-class DeviceDataCollectorTest extends PHPUnit_Framework_TestCase
+class DeviceDataCollectorTest extends TestCase
```

Comment:
tests did not run with PHPUnit 6.3 otherwise: https://travis-ci.org/suncat2000/MobileDetectBundle/jobs/275668491#L538